contracts/src/CrossChainApplications/NativeTokenBridge/INativeTokenDestination.sol

```diff
@@ -0,0 +1,45 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+/**
+ * @dev Interface that describes functionalities for a cross-chain native token bridge.
+ */
+interface INativeTokenDestination {
+    /**
+     * @dev Emitted when tokens are burned in this bridge contract to be bridged to the source chain.
+     */
+    event TransferToSource(
+        address indexed sender,
+        address indexed recipient,
+        uint256 amount,
+        address feeContractAddress,
```

Comment:
do we need to emit the fee cnotract address and fee amount?

Reply:
I guess in Teleporter we emit the whole Teleporter message which includes fee info, is that the thought behind it?

Reply:
Hmm... Maybe we don't need to emit them. The teleporterMessageID should be enough to find the message and get that info, right?

Reply:
Yeah, I tend to agree that the information about the Teleporter message (fee info and message ID) isn't needed as part of the bridge contract events.

Reply:
yup looking back at it the fee info shouldn't be needed for this event